### DISCUSSION TITLE
Mac SE compatibility

### DISCUSSION THREAD

**Discussion by dburr** [erichelgeson/BlueSCSI-v1#129] [Q&A] [answered]
I'm having trouble getting a BlueSCSI 1.0C to work with a Mac SE. It looks like it is receiving power (I see lights and I do get a LOG.TXT on the root of my SD card showing entries for the image file I have on it) but no SCSI devices are detected by HD SC Setup, although a different utility (TattleTech) does show a device present, but shows [weird/garbled information](https://imgur.com/fybi7Cz). I have tried several different image files, including blank ones, and I never see any SCSI devices detected. Termination is set correctly. (single internal device with no external devices) The thing is, I vaguely remember reading a while back on the compatibility page that some SE's are not supported because of an incompatibility with their SCSI chip? But I don't see that information on the compatibility page as it exists today. Has this been proven to be false?

**Comment by erichelgeson** [comment, marked as answer]
What version of the firmware are you using? There was a suspected incompatibility but it could never be proven - most issues had to do with errors in using the wrong scsi driver, or formatting the card, or other normal troubleshooting things. For a while I bought every SE I could find to see if I could reproduce and never could. If you are in the US and if you go through all the troubleshooting I'd gladly trade you one of my many boards for that to see if I could reproduce.

**Reply by dburr** [reply]
> What version of the firmware are you using? There was a suspected incompatibility but it could never be proven - most issues had to do with errors in using the wrong scsi driver, or formatting the card, or other normal troubleshooting things. For a while I bought every SE I could find to see if I could reproduce and never could. If you are in the US and if you go through all the troubleshooting I'd gladly trade you one of my many boards for that to see if I could reproduce.

So... as it turns out, it looks like my BlueSCSI unit itself is at fault. Today I decided to disassemble my other machine (an SE/30 with a BlueSCSI 1.1) so that I can retrobright it, and on a whim decided to try and swap its BlueSCSI into my SE. And to my surprise it booted. So I took the 1.1c that was being problematic in my SE and tried it on the SE/30. And it booted. So it looks like my BlueSCSI is to blame. :-(

I looked at it under the microscope to see if I could find any cracked solder joints, bridges, etc. and everything looks good. So I am at a loss as to what went wrong with it. The only thing I can think of is maybe something went wonky during the firmware upgrade to 20220404? Since I purchased this BlueSCSI several months ago (November 2021 iirc) it was made before the USB firmware came out, so I needed to use the ST-LINK method of updating. Though I don't recall seeing any error or warning messages during the procedure.

I'm sorry for taking your time.

**Reply by erichelgeson** [reply]
No worries at all, glad you got it working!
